packages/flutter/lib/src/widgets/basic.dart

```diff
@@ -1697,6 +1711,8 @@ class Padding extends SingleChildRenderObjectWidget {
 ///
 /// See also:
 ///
+///  * [AnimatedAlign], which automatically transitions the child's
+///    position over a given duration whenever the given [alignment] changes.
 ///  * [CustomSingleChildLayout], which uses a delegate to control the layout of
 ///    a single child.
 ///  * [Center], which is the same as [Align] but with the [alignment] always
```

Comment:
I'd be wary of having too many cross-references. We should probably limit ourselves to 3 or 4 per class, on average. I say this because the other day I totally missed a key cross-reference that I was _looking_ for, because I saw a list of like 7 cross-references and my eyes just glazed over the lot on the assumption that with that many links they'd all be useless.